# Message type to hold a map from another robot
string robot_id
nav_msgs/OccupancyGrid map
